pygotham-2019/videos/1000x-faster-data-manipulation-vectorizing-with-pandas-and-numpy.json

```diff
@@ -0,0 +1,24 @@
+{
+  "copyright_text": "Creative Commons Attribution license (reuse allowed)",
+  "description": "The data transformation code you're writing is correct, but potentially\n1000x slower than it needs to be! In this talk, we will go over multiple\nways to enhance a data transformation workflow with Pandas and Numpy by\nshowing how to replace slower, perhaps more familiar, ways of operating on\nPandas data frames with faster-vectorized solutions to common use cases\nlike:\n\n* if-else logic in applied row-wise functions\n* dictionary lookups with conditional logic\n* Date comparisons and calculations\n* Regex and string column manipulation\n* and others! ...\n\n\nwithout needing a beefier computer, writing Cython, or other libraries\noutside the Pandas ecosystem.",
+  "duration": 1500,
+  "language": "eng",
+  "recorded": "2019-10-05",
+  "related_urls": [
+    "https://2019.pygotham.org/talks/1000x-faster-data-manipulation-vectorizing-with-pandas-and-numpy/"
+  ],
+  "speakers": [
+    "Nathan Cheever"
+  ],
+  "tags": [
+    ""
```

Comment:
You've got empty tag strings in all of the individual talks, this should be an empty list.

Reply:
I can update these files, but this is how the PyGotham 2018 files were generated, too, and those pages seem to generate fine. It also might be worth having `make test` catch this.